runconfig/parse.go

```diff
@@ -513,3 +505,31 @@ func ParseDevice(device string) (DeviceMapping, error) {
 	}
 	return deviceMapping, nil
 }
+
+// ParseVolumesIntoBinds parses any volumes passed by the client and moves any
+// volumes which are in Config.Volumes into HostConfig.Binds.
+func ParseVolumesIntoBinds(c *Config, hc *HostConfig) (*Config, *HostConfig, error) {
+	if c != nil && hc != nil {
+		newVolumes := c.Volumes
+		for bind := range newVolumes {
+			var (
```

Comment:
can remove this block and change the ParseMountSpec below to do the `if` separately.